src/main/java/address/util/FxViewUtil.java

```diff
@@ -24,7 +34,40 @@ public static void applyAnchorBoundaryParameters(Node node, double left, double
         return listView.lookupAll(".scroll-bar")
                 .stream()
                 .filter(foundSb -> ((VirtualScrollBar) foundSb).getOrientation() == Orientation.VERTICAL)
-                .map(obj -> (VirtualScrollBar)obj).findAny();
+                .map(obj -> (VirtualScrollBar) obj).findAny();
+    }
+
+    public static Image getDragView(List<ReadOnlyViewablePerson> draggedPersons) {
```

Comment:
Right. Make sense for this method to be at PersonListCell class, since it is handling the drag and drop operation.